customScripts/expansiontravel.gd

```diff
@@ -118,9 +126,9 @@ func getzonetraveltext(zone,progress):
 		elif captives.size() > 5 && globals.state.reputation.wimborn > 50 && captivebaddiesratio > 69:
 			array.append("A large "+travelgroup()+" of " +travelcheerful()+" "+randomfromarray(['townsfolk','revelers','students','tradesmen','guards'])+" returning to town cheers you and your party along after they see your "+travelcaptivedesc()+" captives shuffling along in their restraints.")
 		if rand_range(0,1) > 0.9 && globals.expansionsettings.sillymode:
-			array.append("You notice a "+travelcheerful()+" "+travelrandomperson()+" "+travelfucking()+" "+travelrandomadj()+" "+travelfucktarget()+" - in broad daylight.\n\n"+travellink()+"\n\nIt's clearly a "+travelreaction()+" moment for him and a "+travelreaction()+" moment for you.")
+			array.append_array(["You notice a "+travelcheerful()+" "+travelrandomperson()+" "+travelfucking()+" "+travelrandomadj()+" "+travelfucktarget()+" - "+travelobserved()+".\n\n"+travellink()+"\n\nIt's clearly a "+travelreaction()+" moment for him and a "+travelreaction()+" moment for you."])
```

Comment:
This does not need to use append_array as there is only a single string

Reply:
comment incorporated on new commit